qt/aqt/mediasrv.py

```diff
@@ -468,31 +433,49 @@ def handle_on_main() -> None:
     return b""
 
 
-def complete_tag() -> bytes:
-    return aqt.mw.col.tags.complete_tag(request.data)
-
-
-def card_stats() -> bytes:
-    args = from_json_bytes(request.data)
-    return aqt.mw.col.card_stats_data(CardId(args["cardId"]))
+# these require a collection
+post_handler_list = [
+    congrats_info,
+    get_deck_configs_for_update,
+    update_deck_configs,
+    next_card_states,
+    set_next_card_states,
+    change_notetype,
+]
+
+
+exposed_backend_list = [
+    # I18nService
+    "i18n_resources",
+    # NotesService
+    "get_note",
+    # NotetypesService
+    "get_notetype_names",
+    "get_change_notetype_info",
+    # StatsService
+    "card_stats",
+    "graphs",
+    "get_graph_preferences",
+    "set_graph_preferences",
+    # TagsService
+    "complete_tag",
+]
+
+
+def snakecase_to_camelcase(name: str) -> str:
+    [first, *rest] = name.split("_")
+    return first + "".join([r.capitalize() for r in rest])
+
+
+def access_backend(endpoint: str) -> Callable[[], bytes]:
+    return lambda: getattr(aqt.mw.col._backend, f"{endpoint}_raw")(request.data)
```

Comment:
I thought of this as a way to obscure that we're using `col._backend` directly.
Combined with using the `protobufjs` services in `lib/proto.js` it also makes sure that we don't have small changes in API names (e.g. `graphData` in JS calls `getGraphData` in Python calls `graphsData` in Protobuf).

Reply:
I like that it cuts down on boilerplate and inconsistencies. For the name, how about something like 'raw_backend_request' or 'direct_backend_request'?